org.eclipse.buildship.core.test/src/main/groovy/org/eclipse/buildship/core/launch/internal/GradleClasspathProviderUpdateTest.groovy

```diff
@@ -32,7 +32,7 @@ class GradleClasspathProviderUpdateTest extends ProjectSynchronizationSpecificat
         importAndWait(projectDir)
 
         then:
-        hasGradleClasspathProvider(launchConfiguration)
+        waitFor { hasGradleClasspathProvider(launchConfiguration) }
```

Comment:
⭕️ Why did all the tests change in this way? Shouldn't importAndWait take care of the waiting?